ddtrace/tracer/option.go

```diff
@@ -111,6 +111,9 @@ var (
 
 	// defaultMaxTagsHeaderLen specifies the default maximum length of the X-Datadog-Tags header value.
 	defaultMaxTagsHeaderLen = 128
+
+	// defaultRateLimit specifies the default trace rate limit used when DD_TRACE_RATE_LIMIT is not set.
+	defaultRateLimit = 100.0
```

Comment:
I would _LOVE_ it if that also explained what unit this is expressed in... or something... Because just `100.0` doesn't quite tell me a lot.

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/3033/commits/4bb0581fe708ab53f71e56c4ac89f2e067526ffe